charts/central-viewer/templates/ingress.yaml

```diff
@@ -1,8 +1,12 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "central-viewer.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-apiVersion: traefik.containo.us/v1alpha1
-kind: IngressRoute
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Nice! Default k8s networking, right?

Reply:
Yes, de default inderdaad, dit is ook wat `helm create` standaard als template genereert. Nu is het wel zo dat `networking.k8s.io/v1beta1` eigenlijk al deprecated is en verwijderd wordt vanaf Kubernetes v1.22. De nieuwe `networking.k8s.io/v1` (Kube v1.19+) gebruikt iets andere syntax en vraagt dus aanpassingen aan de Ingress om beide te kunnen ondersteunen. Ik wilde nog even wachten op nieuwe versies van de Helm CLI waarin dat voor ons gescaffolt wordt, anders moeten we dat op termijn zelf toevoegen.